wallet/src/de/schildbach/wallet/ui/username/voting/RequestUsernameFragment.kt

```diff
@@ -91,4 +117,22 @@ class RequestUsernameFragment : Fragment(R.layout.fragment_request_username) {
     private fun hideKeyboard() {
         KeyboardUtil.hideKeyboard(requireContext(), binding.usernameInput)
     }
+
+    private fun showErrorDialog() {
+        val dialog = AdaptiveDialog.create(
+            R.drawable.ic_error,
+            getString(R.string.something_wrong_title),
+            getString(R.string.there_was_a_network_error),
+            getString(R.string.close),
+            getString(R.string.try_again)
+
+        )
+        dialog.show(requireActivity()) {
+            if (it == true) {
+                requestUserNameViewModel.submit()
+            } else {
+                dialog.dismiss()
```

Comment:
I don't think `dialog.dismiss()` this is needed. The dialog will dismiss itself.

Reply:
removed